### PR TITLE
HOSTEDCP-938: Added PSA default profile to RunTimeDefault in operator deployment

### PR DESCRIPTION
**Pull request by jparrill**
**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story*:
Fixes #[HOSTEDCP-938](https://issues.redhat.com/browse/HOSTEDCP-938)

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [x] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

**Comment by jparrill** [comment]
area/hypershift-operator

File: cmd/install/assets/hypershift_operator.go [hunk 1]
```diff
@@ -38,6 +38,12 @@ const (
 	DefaultPriorityClass = "hypershift-control-plane"
 )
 
+var (
+	// allowPrivilegeEscalation is used to set the status of the
+	// privillegeEscalation on SeccompProfile
```
Comment (bryan-cox) [inline cmd/install/assets/hypershift_operator.go:43]:
```suggestion
	// privilegeEscalation on SeccompProfile
```

**Review by bryan-cox — COMMENTED** [review]
One minor nit

**Comment by bryan-cox** [comment]
/retitle HOSTEDCP-938: Added PSA default profile to RunTimeDefault in operator deployment
/area hypershift-operator

**Pull request merged**